src/SprykerSdk/Spryk/SprykFacade.php

```diff
@@ -18,14 +18,14 @@ class SprykFacade implements SprykFacadeInterface
 
     /**
      * @param string $sprykName
-     * @param string[] $includeOptionalSubSpryks
      * @param \SprykerSdk\Spryk\Style\SprykStyleInterface $style
+     * @param array $options
      *
      * @return void
      */
-    public function executeSpryk(string $sprykName, array $includeOptionalSubSpryks, SprykStyleInterface $style): void
+    public function executeSpryk(string $sprykName, SprykStyleInterface $style, array $options): void
```

Comment:
IMO `$sprykName` and `$options` belong together. I think that these two arguments are required but the output could be optional. I would like to keep the order of arguments.